SofaKernel/modules/SofaBaseMechanics/SofaBaseMechanicsConfig.cmake.in

```diff
@@ -0,0 +1,13 @@
+# CMake package configuration file for the @PROJECT_NAME@ module
+
+@PACKAGE_GUARD@
+@PACKAGE_INIT@
+
+find_package(SofaEigen2Solver REQUIRED)
+find_package(SofaBaseTopology REQUIRED)
+
```

Comment:
Missing `find_package(SofaBaseLinearSolver REQUIRED)` ?

Reply:
I did not set it 'cause SofaEigen2Solver has also the dependency on SofaBaseLinearSolver.
But I can add it if you want

Reply:
It's just to be consistent with the CMakeFile.txt.

Also, since some components are actually using it, and if at some point SofaBaseMechanics doesn't depend anymore on SofaEigen2Solver, at least this requirement will still be there and valid.